packages/vetrina/src/Tracking.purs

```diff
@@ -0,0 +1,14 @@
+module Tracking where
+
+import Data.Function.Uncurried (Fn3, runFn3)
+import Data.Maybe (Maybe(..))
+import Effect (Effect)
+import KSF.User (OrderNumber)
+import Prelude (Unit, pure, unit)
+
+foreign import data Tracker :: Type
+foreign import transaction_ :: Fn3 OrderNumber String String (Effect Unit)
+
+transaction :: OrderNumber -> Maybe String -> Maybe String -> Effect Unit
+transaction ord (Just t) (Just p)  = runFn3 transaction_ ord t p
```

Comment:
Instead of calling these `t` and `p`, let's make up something more meaningful.

Reply:
`ord` could also be better (this would imply it's maybe an order of some sorts?), however one can see from the type signature it's an `OrderNumber` so it's fine-ish 🙂 The other ones are just `Strings` so I have no idea what they are.